tests/components/homematicip_cloud/test_climate.py

```diff
@@ -410,27 +468,36 @@
         blocking=True,
     )
     assert home.mock_calls[-1][0] == "activate_absence_with_period"
-    assert home.mock_calls[-1][1] == (datetime.datetime(2019, 2, 17, 14, 0),)
+    assert home.mock_calls[-1][1] == (datetime.datetime(2019, 2, 17, 14, 0), )
     assert len(home._connection.mock_calls) == 4  # pylint: disable=protected-access
 
     await hass.services.async_call(
         "homematicip_cloud",
         "activate_vacation",
-        {"endtime": "2019-02-17 14:00", "temperature": 18.5, "accesspoint_id": HAPID},
+        {
+            "endtime": "2019-02-17 14:00",
+            "temperature": 18.5,
+            "accesspoint_id": HAPID
+        },
         blocking=True,
     )
     assert home.mock_calls[-1][0] == "activate_vacation"
-    assert home.mock_calls[-1][1] == (datetime.datetime(2019, 2, 17, 14, 0), 18.5)
+    assert home.mock_calls[-1][1] == (datetime.datetime(2019, 2, 17, 14,
+                                                        0), 18.5)
     assert len(home._connection.mock_calls) == 5  # pylint: disable=protected-access
 
     await hass.services.async_call(
         "homematicip_cloud",
         "activate_vacation",
-        {"endtime": "2019-02-17 14:00", "temperature": 18.5},
+        {
+            "endtime": "2019-02-17 14:00",
+            "temperature": 18.5
+        },
         blocking=True,
     )
     assert home.mock_calls[-1][0] == "activate_vacation"
-    assert home.mock_calls[-1][1] == (datetime.datetime(2019, 2, 17, 14, 0), 18.5)
+    assert home.mock_calls[-1][1] == (datetime.datetime(2019, 2, 17, 14,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4570914)